roles/common/tasks/main.yml

```diff
@@ -7,6 +7,8 @@
     - empty.yml
   tags:
     - vars
+    # We need these vars for the entitlements tag to work
+    - entitlements
```

Comment:
I'm confused.

I'm adding the entitlements tag so I can entitle machines as a separate step if I need to. I don't necessarily want this to run *always*. The vars tag was added in 5eece205, by you.